package.json

```diff
@@ -163,10 +163,11 @@
     "prepublishOnly": "npm run build-lib:prod",
     "clean": "rm -r dist-demo dist",
     "start": "node scripts/start-demo.js",
-    "test": "concurrently --success first --kill-others \"http-server src/loaders/fixtures/ --cors=\"*\" -s \" \"karma start scripts/karma.config.js --single-run\"",
-    "test:watch": "concurrently --kill-others \"http-server src/loaders/fixtures/ --cors=\"*\" -s \" \"karma start scripts/karma.config.js --auto-watch\"",
-    "cypress:open": "concurrently --kill-others \"http-server src/loaders/fixtures/ --cors=\"*\" -s \" \"cypress open\"",
-    "cypress:run": "concurrently --success first --kill-others \"http-server src/loaders/fixtures/ --cors=\"*\" -s \" \"cypress run\"",
+    "start:fixtures": "http-server src/loaders/fixtures/ --cors=\"*\" -s",
+    "test": "concurrently --success first --kill-others \"npm run start:fixtures\" \"karma start scripts/karma.config.js --single-run\"",
+    "test:watch": "concurrently --kill-others \"npm run start:fixtures\" \"karma start scripts/karma.config.js --auto-watch\"",
+    "cypress:open": "concurrently --kill-others \"npm run start:fixtures\" \"cypress open\"",
+    "cypress:run": "concurrently --success first --kill-others \"npm run start:fixtures\" \"cypress run --headed --browser chrome\"",
```

Comment:
The change is to add `npm run start:fixtures` before the other commands? I really don't know the npm toolchain that well, but would it be of any use to set up a [`pre`](https://docs.npmjs.com/cli/v6/using-npm/scripts#pre--post-scripts) action for all of these? ... All of these `pre`s would be the same, but it would make it easier to see at a glance. Or maybe pull out the shared start to these commands on, and then use [`--`](https://stackoverflow.com/a/27765622) with just the arguments that are different?

Reply:
Agreed, the issue is the `concurrently` that is managing them though.  We need that ability to stop the servers/start them simultaneously.  I'll try to find a better solution.

Reply:
i.e We can't just start them independently.